### PR TITLE
Adds a button that allows admins to activate the gateway.

### PR DESCRIPTION
**Pull request by theOperand**
also fixes the away missions to the best of my ability. im going to need some help
also adds a config option to load away missions at roundstart. starts off.
if you dont want admins to load away missions just re-comment all the files in filelist.txt

currently away missions are fucked on account of being outdated by several months.
- [x] Lightning is completely fucked, everything has max brightness @MrPerson please tell me how to fix this. (the light objects are missing) _Thanks mrperson_
- [ ] ~~Tons of locker-related runtimes, I need somebody to use a regex to clean out all those obsolete vars.~~ _I DID IT MYSELF (because mso is mean :V). You may have to re-do your lockers, @WJohn_ _E2: whoops all the lockers are invisible now_ _E3: yeah all the lockers need some love_
- [x] Air doesn't work either and gives a lot of `cannot read null.oxygen` runtimes. @Aranclanos any idea? _had to initialize everything and shit_

`icon_(broken|closed|locked|off|opened) = "(\w+)";`

Now the lockers dont runtime anymore, here's the runtime for the air:

```
runtime error: Cannot read null.oxygen
proc name: compare (/datum/gas_mixture/compare)
  source file: gas_mixture.dm,910
  usr: null
  src: /datum/gas_mixture (/datum/gas_mixture)
  call stack:
/datum/gas_mixture (/datum/gas_mixture): compare(null)
the floor (64,125,8) (/turf/simulated/floor/plasteel): process cell()
Air (/datum/subsystem/air): process active turfs()
Air (/datum/subsystem/air): fire()
/datum/controller/game_control... (/datum/controller/game_controller): process()
```

All of the other atmos machinery runtimes, too.

### CODE REVIEW

File: code/modules/admin/verbs/gateway.dm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		return
+
+	log_admin("[usr] has loaded the away mission [mapname]")
+	message_admins("[usr] has loaded the away mission [mapname]")
+	loadAwayMission(mapname, 1)
+
+	var/obj/machinery/gateway/centerstation/gateway = locate() in world //cri
+	var/obj/machinery/gateway/centeraway/gatewayaway = locate() in world //double cri
+	gateway.wait = -1
+	gateway.awaygate = gatewayaway
```
Comment (Miauw62) [inline code/modules/admin/verbs/gateway.dm]: :^]

**Comment by Miauw62** [comment]
@WJohn wtf why aren't the personal lockers on your map actually personal lockers instead of renamed normal lockers???

**Comment by WJohn** [comment]
I made them regular secure lockers and then changed their sprites, names, descriptions, and accesses. I haven't updated any of these things in a while so there's bound to be some major issues.

**Comment by Miauw62** [comment]
Oh, I see.
Yeah, I think all of that is borked now :x 

Wuh, I need a way to call initialize() on the atom_movables, but I have no idea how.

**Comment by Fayrik** [comment]
Should gasmixes even be null-able? Could we not just hack that problem away by making a quick if (!gasmix) gasmix = vacuum?

**Comment by Supermichael777** [comment]
no, sounds like air should be there

**Comment by Miauw62** [comment]
nothing was initialized on the new zs, not lightning, not pipenets, not air, etc

**Comment by Miauw62** [comment]
it werks.

@WJohn are the asteroid tiles on moon outpost supposed to contain tons of CO2 instead of vacuum?

**Comment by WJohn** [comment]
Yes, both are meant to have custom atmospheres, MO19 is supposed to have enough CO2 and oxygen for you to choke without any warning come up. You'll still gasp though.

UO45 on the other hand is supposed to have tons of hot plasma and other gasses that are just beneath ignition temperature. A little oxygen from the outpost, through a breach, and a tiny bit of extra heat can easily make a self sustainig fire.

**Comment by RemieRichards** [comment]
@Miauw62 ur a poop

**Comment by Miauw62** [comment]
i dont care enough about away missions to fix them

**Comment by Miauw62** [comment]
also you have more PRs in purgatory than me :V

**Comment by Razharas** [comment]
Purgatory just got cleansed be the holy fires of codequisition
You are next miauw

**Comment by Miauw62** [comment]
;_;

(i should probably remake this PR entirely but first I have to debug ai tracking)

**Pull request closed without merging**